examples/javascript/index.html

```diff
@@ -1,37 +1,40 @@
 <!doctype html>
 <html>
+
```

Comment:
nit: Are we planning to add extra lines in all the other examples? Otherwise, remove for consistency